### PR TITLE
Make Hpr Number optional

### LINKED ISSUE #429: Make HprNumber optional
It is currently mandatory, even if you can remove it from the config, it will still be added.  

We make it optional based on the config, and then you don't need to have it enabled in

### PR DESCRIPTION
**Pull request by OsirisTerje**
Fixes #429

### CODE REVIEW

File: Fhi.HelseId.Web/HelseIdWebKonfigurasjon.cs [hunk 1]
```diff
@@ -47,9 +47,12 @@ protected override IEnumerable<string> FixedScopes
                 "profile",
                 "helseid://scopes/identity/pid",
                 "helseid://scopes/identity/pid_pseudonym",
-                "helseid://scopes/identity/security_level",
-                "helseid://scopes/hpr/hpr_number"
+                "helseid://scopes/identity/security_level"
             };
+            if (UseHprNumber)
```
Comment (martinmine) [inline Fhi.HelseId.Web/HelseIdWebKonfigurasjon.cs:52]: Do we know if we have any consumers who is using HPR number and has this disabled? In those cases we can argue that is a bug, but I want to nevertheless bring this up to highlight it.

Comment (OsirisTerje) [reply Fhi.HelseId.Web/HelseIdWebKonfigurasjon.cs:52]: Yes, we do.  The complaint came from a dev.

File: Fhi.HelseId.Tests/HelseIdWebKonfigurasjonTests.cs [hunk 2]
```diff
@@ -20,7 +20,7 @@ public void HelseIdWebConfigurationTest()
                 Assert.That(sut.AuthUse, Is.True, "2");
                 Assert.That(sut.UseHttps, Is.True, "3");
                 Assert.That(sut.UseHprNumber, Is.False, "4");
-                Assert.That(sut.AllScopes, Has.Count.GreaterThan(7), "5");
+                Assert.That(sut.AllScopes, Has.Count.GreaterThan(6), "5");
```
Comment (martinmine) [inline Fhi.HelseId.Tests/HelseIdWebKonfigurasjonTests.cs:23]: Should we also add a test when UseHprNumber is true, that this is set to 7?

Comment (OsirisTerje) [reply Fhi.HelseId.Tests/HelseIdWebKonfigurasjonTests.cs:23]: Agree, will do

**Pull request merged**